CppCoreGuidelines.md

```diff
@@ -8871,7 +8871,7 @@ Convenience of use and avoidance of errors.
     Day today = Day::sat;
     Day tomorrow = ++today;
 
-The use of a `static_cast` is not pretty, but
```

Comment:
This changes the meaning -- please leave this one as is, and similarly in 11025 and 11909,

Reply:
reverted